store/tikv/gcworker/gc_worker.go

```diff
@@ -888,6 +894,11 @@ func (w *GCWorker) resolveLocksForRange(ctx context.Context, safePoint uint64, s
 			continue
 		}
 
+		bo = tikv.NewBackoffer(ctx, tikv.GcResolveLockMaxBackoff)
+		failpoint.Inject("setGcResolveMaxBackoff", func(v failpoint.Value) {
+			sleep := v.(int)
+			bo = tikv.NewBackoffer(ctx, sleep)
+		})
```

Comment:
Can it be moved to below line 916? So the last loop will go to break before creating a useless backoffer.